server/src/main/java/org/elasticsearch/search/profile/ProfileResult.java

```diff
@@ -49,26 +51,25 @@
  * "children" queries if applicable
  */
 public final class ProfileResult implements Writeable, ToXContentObject {
-
     static final ParseField TYPE = new ParseField("type");
     static final ParseField DESCRIPTION = new ParseField("description");
+    static final ParseField BREAKDOWN = new ParseField("breakdown");
     static final ParseField NODE_TIME = new ParseField("time");
     static final ParseField NODE_TIME_RAW = new ParseField("time_in_nanos");
     static final ParseField CHILDREN = new ParseField("children");
-    static final ParseField BREAKDOWN = new ParseField("breakdown");
 
     private final String type;
     private final String description;
-    private final Map<String, Long> timings;
```

Comment:
This is the change the breaks BWC. I'm not 100% sure it is ok. I'm hunting.

The effects will be that versions of the high level rest client that don't have this won't be able to read breakdowns that include non-numbers. They'll blow up at:
```
ensureExpectedToken(XContentParser.Token.VALUE_NUMBER, parser.nextToken(), parser::getTokenLocation);
```

Reply:
Now that I think about this I'm going to move the "extra" stuff into an "extra" field and leave the breakdown all longs. I think that preserves backwards compatibility and is less confusing.